bot.py

```diff
@@ -97,18 +126,20 @@ def is_friend(member):
 async def on_voice_state_update(before, after):
     global elevator
     global in_elevator
+    global current_track
     voice_channel = discord.utils.get(after.server.channels, name=elevatorName)
     if in_elevator and after.id == client.user.id and after.voice.voice_channel != voice_channel:
         await client.move_member(after, voice_channel)
     elif in_elevator and len(voice_channel.voice_members) == 1:
         elevator.stop()
         elevator = None
         await client.voice_client_in(after.server).disconnect()
+        current_track = ELEVATOR_MUSIC
```

Comment:
This seems like an odd decision. If we've changed the song wouldn't we want it to stay changed?